t/t0014-alias.sh

```diff
@@ -37,4 +37,15 @@ test_expect_success 'looping aliases - internal execution' '
 #	test_i18ngrep "^fatal: alias loop detected: expansion of" output
 #'
 
+GIT_TRACE=1 && export GIT_TRACE
+
+test_expect_success 'run-command parses empty args properly, using sq_quote_buf_pretty' '
+	cat >expect <<-EOF &&
+	trace: resolved executable dir: C:/gfw
```

Comment:
`C:/gfw` doesn't seem like it will be correct for everyone.

Reply:
Maybe drop the trace stuff? Doesn't seem like you need it.

Reply:
//facepalm!